composer.json

```diff
@@ -20,15 +20,15 @@
         }
     ],
     "require": {
-        "php": ">=7.4"
+        "php": "^7.2 || ^8.0"
```

Comment:
Most Aura packages uses `>=` style. It intentionally block PHP 9, is that okay?

Reply:
Yes, i am intentionally blocking 9.

We don't know if it can support 9. Only if it supports we can add